packages/webapp/queries/base/requests/proposalsQuery.ts

```diff
@@ -0,0 +1,67 @@
+import * as Sentry from '@sentry/nextjs'
+
+import { getProposalState } from '@data/contract/requests/getProposalState'
+import { BaseSDK } from '@queries/resolvers.ts'
+import { CHAIN_IDS } from '@constants/types.ts'
+import { Proposal } from './proposalQuery'
+
+export interface ProposalsResponse {
+  proposals: Proposal[]
+  pageInfo?: {
+    hasNextPage: boolean
+  }
+}
+
+export const getProposals = async (
+  chainId: number,
+  token: string,
+  limit: number = 100,
+  page?: number
+): Promise<ProposalsResponse> => {
+  try {
+    const data = await BaseSDK.connect().proposals({
+      where: {
+        dao: token.toLowerCase(),
```

Comment:
there a reason we lowercase this instead of checksum?

Reply:
Builder